app/models/eligibilities/in_sam.rb

```diff
@@ -0,0 +1,7 @@
+module Eligibility
+  class InSam
+    def eligible?(user)
```

Comment:
I think I'd prefer to clarify eligibility in terms of actions like `user_can_bid?`